lib/node_modules/@stdlib/math/base/special/acoth/test/fixtures/julia/runner.jl

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env julia
+#
+# @license Apache-2.0
+#
+# Copyright (c) 2018 The Stdlib Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import JSON
+
+"""
+    gen( x, name )
+
+Generate fixture data and write to file.
+
+# Arguments
+
+* `x`: domain
+* `name::AbstractString`: output filename
+
+# Examples
+
+``` julia
+julia> x = linspace( -1000, 1000, 2001 );
+julia> gen( x, \"data.json\" );
+```
+"""
+function gen( x, name )
+	y = acoth( x );
+
+	# Store data to be written to file as a collection:
+	data = Dict([
+		("x", x),
+		("expected", y)
+	]);
+
+	# Based on the script directory, create an output filepath:
+	filepath = joinpath( dir, name );
+
+	# Write the data to the output filepath as JSON:
+	outfile = open( filepath, "w" );
+	write( outfile, JSON.json(data) );
+	close( outfile );
+end
+
+# Get the filename:
+file = @__FILE__;
+
+# Extract the directory in which this file resides:
+dir = dirname( file );
+
+# Positive medium values:
+x = linspace( 1.0, 3.0, 500 );
```

Comment:
```suggestion
x = linspace( 1.0, 3.0, 503 );
```

Avoid nice even array lengths when generating fixtures. Otherwise, one generates predictable bit patterns, which may mask implementation flaws.

Reply:
Nice! Good hint.